### PR TITLE
Migrate to Next.js

### PR DESCRIPTION
**Pull request by Kuinox**

### CODE REVIEW

File: src/app/layout.tsx
Comment (LPeter1997) [inline src/app/layout.tsx:1]: Why doesn't the file name follow the convention? Rest are named after the functional component they encapsulate

File: src/app/docs/page.tsx
Comment (LPeter1997) [inline src/app/docs/page.tsx:1]: Soft-convention break here too because of casing

File: src/app/community/page.tsx
Comment (LPeter1997) [inline src/app/community/page.tsx:1]: Same here

File: src/app/layout.tsx
Comment (Kuinox) [reply src/app/layout.tsx:1]: The next.js routing uses the names of the files.

**Review by LPeter1997 — APPROVED** [review]

**Pull request merged**